docs/kyma/04-04-cluster-installation.md

```diff
@@ -246,74 +243,26 @@ This installation guide explains how you can quickly deploy Kyma on a cluster wi
    >* [Istio production profile](/components/service-mesh/#configuration-service-mesh-production-profile)
    >* [OAuth2 server production profile](/components/security/#configuration-o-auth2-server-profiles)
 
-1. Deploy Kyma:
-
-    ```bash
-    kubectl apply -f https://github.com/kyma-project/kyma/releases/download/$KYMA_VERSION/kyma-installer-cluster.yaml
-    ```
-
-2. Check if the Pod of the Kyma Installer is running:
-
-    ```bash
-    kubectl get pods -n kyma-installer
-    ```
-
-3. To watch the installation progress, run:
+1. Install Kyma using Kyma CLI:
 
     ```bash
-    while true; do \
-      kubectl -n default get installation/kyma-installation -o jsonpath="{'Status: '}{.status.state}{', description: '}{.status.description}"; echo; \
-      sleep 5; \
-    done
+    kyma install -s $KYMA_VERSION
```

Comment:
But this is an environment variable being set in one of the previous steps. The user can just run `kyma install -s $KYMA_VERSION` and it will be resolved to the version he set before.

Reply:
ok, then it makes sense :)